From akpm@linux-foundation.org  Thu Jun 19 11:54:05 2008
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Mon, 09 Jun 2008 16:39:51 -0700
Subject: uwb: fix Kconfig causing undefined references
To: greg@kroah.com
Cc: linux-usb@vger.kernel.org, akpm@linux-foundation.org, randy.dunlap@oracle.com
Message-ID: <200806092339.m59Ndquq015001@imap1.linux-foundation.org>


From: Randy Dunlap <randy.dunlap@oracle.com>

Fix UWB kconfig so that function reference problems are not generated.

next-20080430/drivers/uwb/whc-rc.c:478: undefined reference to `hwarc_filter_event'
next-20080430/drivers/uwb/whc-rc.c:478: undefined reference to `hwarc_filter_cmd'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/uwb/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/uwb/Kconfig
+++ b/drivers/uwb/Kconfig
@@ -44,6 +44,7 @@ config UWB_HWA
 config UWB_WHCI
         tristate "UWB Radio Control driver for WHCI-compliant cards"
         depends on PCI
+	depends on UWB_HWA
         help
           This driver enables the radio controller for WHCI cards.
 
